src/PlatformHelpers.web.js

```diff
@@ -1,9 +1,3 @@
-export const Linking = {
-  addEventListener: () => {},
-  removeEventListener: () => {},
-  getInitialURL: () => Promise.reject('Unsupported platform'),
-};
+import { BackHandler } from 'react-native';
```

Comment:
I know that everybody rewrites `react-native` into `react-native-web` in the webpack config, but it just feels so hacky to be apparently importing mobile code from a web module </rant>

Reply:
Why keep this file? It does the same thing as the other one. Isn't renaming `PlatformHelpers.native.js` to `PlatformHelpers.js` should address the issue?